neusomatic/python/preprocess.py

```diff
@@ -21,19 +21,22 @@
 from utils import concatenate_vcfs
 
 FORMAT = '%(levelname)s %(asctime)-15s %(name)-20s %(message)s'
```

Comment:
The format and config should only happen if is used as a script. So, under the `if __name__ == "__main__":` block.